xml/System.Data.Spatial/DbGeometry.xml

```diff
@@ -1135,8 +1135,8 @@
         <ReturnType>System.Nullable&lt;System.Double&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the Measure (M coordinate) of this DbGeometry value, if it represents a point. &lt;returns&gt;The Measure (M coordinate) value of this geometry value, if it represents a point; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <value>The Measure (M coordinate) of this DbGeometry value.</value>
+        <summary>Gets the Measure (M coordinate) of this DbGeometry value, if it represents a point.</summary>
+        <value>The Measure (M coordinate) value of this geometry value, if it represents a point; otherwise <see langword="null" />.</value>
```

Comment:
```suggestion
        <value>The Measure (M coordinate) value of this geometry value, if it represents a point; otherwise, <see langword="null" />.</value>
```